build.properties.dist

```diff
@@ -177,10 +177,19 @@ phpcs.extensions = php inc module install info test profile theme css js
 # The default configuration file to generate.
 phpcs.config = ${project.basedir}/phpcs.xml
 
-# The coding standard to use.
-phpcs.standard = ${project.basedir}/phpcs-ruleset.xml
+# The default path of qa-automation.
+phpcs.qaautomation.path = ${project.basedir}/vendor/ec-europa/qa-automation
 
-# Paths to check, delimited by semicolons.
+# The locations for installed standards, delimited by spaces, commas or semicolons.
+phpcs.installed.paths = ${phpcs.qaautomation.path}/phpcs/SubStandards
+
+# The coding standards to use, delimited by spaces, commas or semicolons.
+phpcs.standards = Platform;${project.basedir}/phpcs-ruleset.xml
+
+# Returns a 0 error code when only warnings are found if enabled. Ment for CI.
```

Comment:
And you need to document possible setting values, see my previous comment.
For instance "0 is disabled, 1 is enabled."